### ISSUE TITLE
consolidate comms channels

### ISSUE THREAD

**Issue by berezovskyi** [eclipse/lyo#6]
This issue breaks out a chat discussion for the preferred Lyo communication channels. Here is a summary:

* We have the lyo-dev mailing list, we are moving to make it a dev-friendly list, not the place for users.
* I have added a bit of insult to the injury by asking the webmaster to create a lyo-users mailing list.
* We have announced OSLC community forum around the OSLCfest, which has a Lyo forum under it.
* Finally, we still have the Lyo forum under Eclipse Forums.

Proposal is to:

1. Ask the webmaster to close down the lyo-users list due to a lack of activity.
2. Post a big announcement on the Lyo forum under Eclipse about the move to open-services.net

**Comment by berezovskyi** [comment]
@jadelkhoury @jamsden @yashkhatri what's your thoughts?

**Comment by berezovskyi** [comment]
https://bugs.eclipse.org/bugs/show_bug.cgi?id=538426 is the bug for creating lyo-users, we may want to reopen it and ask to nuke it

**Comment by berezovskyi** [comment]
Also, there were multiple ideas for chats:

* Slack (in use, requires an invite app)
* Gitter (in use, no activity)
* Zulip (was in use, replaced by the links to Gitter)
* Discord (not in use, just squatted a handle)
* IRC (not in use, channels #oslc and #lyo do not seem to be in use)

**Comment by berezovskyi** [comment]
I think I posted a link before but let me do it again:

https://forum.open-services.net/c/sdks/lyo

Stackoverflow has very strict rules on the quality of questions. But I do monitor any OSLC questions there.

I am actually fine not closing any channels, I get email notifications from any of them so it does not add any overhead for me.

--
/Andrew

On 24 Nov 2018, at 22:33, Jad El-khoury <notifications@github.com<mailto:notifications@github.com>> wrote:


is there an forum featue on open-services.net<http://open-services.net>? if so, I have no opinion on moving from eclipse forum.

I found Slack an odd choice for an individual that just wants to post a question. Such an overkill to ahve to join teh Slack OSLC community for that. Besides that, I don't have much opinion/experience with the other chat alternatives.

It would be nice with a forum, since teh posted questions are at least reachable when someone does a google search. Having said that, have we considered stackoverflow?
